frontend-react/src/utils/Analytics.ts

```diff
@@ -0,0 +1,20 @@
+import { IEventTelemetry } from "@microsoft/applicationinsights-web";
+
+import { getAppInsights } from "../TelemetryService";
+
+export enum EventName {
```

Comment:
Could we instead rely on `FeatureType` to provide the names of features and `EventName` to provide the event name itself, and mash them together in a string? If we change `FeatureType.DAILY_DATA`, for example, we'd want to ensure we are getting the right `Feature | Event` string

Reply:
That was my intent initially :)

Reply:
Should this be named `PageName` instead of `FeatureType` and split up `FeatureType.ADMIN` then?  It looks like the intention is to organize them by page, in which case 'feature' feels a little vague.

Reply:
Penny and I chatted about this yesterday, but for posterity:

My thought here was that features can exist across pages so having the insight into "feature A being used on page 1" versus "feature A being used on page 2" might be helpful.  But this isn't a blocking point in my mind, and we can address it in the future if we ever need to!